app.rb

```diff
@@ -0,0 +1,121 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_teacher
+    print "teacher's specialization: "
+    specialization = gets.chomp
+    print "teacher's age: "
+    age = gets.chomp
+    print "teacher's name: "
+    name = gets.chomp
+    teacher = Teacher.new(specialization, age, name, parent_permission: true)
+    @people << teacher
+    puts 'You have successfully registered a Teacher'
+  end
```

Comment:
- [ ] Kindly note that the teacher's attributes are misplaced. In place on `name`, teacher's `age` is displaying, and in place of `age`, I can see `specialization` ⚠Also, teacher's `name` is not present. Kindly fix these issues getting the expected behaviour of app as given in the project requirements 🤓⛏

  ![image](https://github.com/abeltsew/oop-school-library/assets/106119105/0634c7f2-fc18-4b73-a57c-1e651e7f4fa4)